great_expectations/data_context/data_context/ephemeral_data_context.py

```diff
@@ -0,0 +1,15 @@
+from great_expectations.data_context.data_context.abstract_data_context import (
+    AbstractDataContext,
+)
+
+
+class EphemeralDataContext(AbstractDataContext):
```

Comment:
@Shinnnyshinshin I feel that stylistically this does not read as I would imagine.  When something "Abstract" is defined, one expects there to be abstract interface methods declared, meaning that the descendants would need to implement them.  We do not seem to have this structure here; perhaps we should deliberate some more?  Thanks.